src/SignalR/clients/ts/signalr/package.json

```diff
@@ -46,17 +47,17 @@
     "src/**/*"
   ],
   "devDependencies": {
-    "@types/eventsource": "^1.0.2",
-    "@types/jest": "^23.3.2",
-    "@types/node": "^10.9.4",
-    "@types/tough-cookie": "^2.3.6",
-    "es6-promise": "^4.2.2"
-  },
-  "dependencies": {
+    "@types/eventsource": "^1.1.5",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^14.14.31",
+    "@types/tough-cookie": "^4.0.0",
+    "process": "^0.11.10",
     "eventsource": "^1.0.7",
```

Comment:
These need to be dependencies for Node apps to be able to use the library without bringing in polyfills manually.

Reply:
Done